### PR TITLE
DO NOT MERGE: Update CKEDITOR to v4.11.3

### PR DESCRIPTION
**Pull request by julien**
This is a first draft for the update of CKEDITOR provided
by the liferay-ckeditor npm package.

### CODE REVIEW

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/build.gradle [hunk 1]
```diff
@@ -2,18 +2,13 @@ import com.liferay.gradle.util.FileUtil
...
 task buildCKEditorBBCode(type: ConcatenateTask)
+task buildCKEditorPlugins(type: Copy)
 task buildCKEditorScayt(type: Copy)
 task buildCKEditorWsc(type: Copy)
 
 String ckEditorVersion = "4.6.0"
-String ckEditorVersionLfr = "4.6.0.6"
-
 String ckEditorScaytUrl = "https://download.ckeditor.com/scayt/releases/scayt_${ckEditorVersion}.zip"
 String ckEditorWscUrl = "https://download.ckeditor.com/wsc/releases/wsc_${ckEditorVersion}.zip"
```
Comment (jbalsas) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/build.gradle]: This should probably be updated, shouldn't it? We should at least grab the latest versions

Comment (julien) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/build.gradle]: Yes, that's a mistake sorry

**Comment by julien** [comment]
So just leaving some notes here, even though this PR is not ready for review
- We still need to publish liferay-ckeditor to the npm registry
- I found a way to build a non minified version of ckeditor (I need to see a few details with @wincent)

**Comment by jbalsas** [comment]
Closing in favour of #1741

**Pull request closed without merging**